src/main/java/com/google/api/codegen/util/SampleConfigSanitizer.java

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.util;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.yaml.snakeyaml.Yaml;
+
+public class SampleConfigSanitizer {
+
+  private SampleConfigSanitizer() {
+    // utility class
+  }
+
+  private static final String TYPE_FIELD_NAME = "type";
+  private static final String SCHEMA_VERSION_FIELD_NAME = "config_schema_version";
```

Comment:
We resolved to use `schema_version` instead of `config_schema_version`. IIRC, we don't need `config_schema_version` for anything.

Reply:
test samples in `gapic-generator` use `config_schema_version`...?

Reply:
At some point there was a discussion that we should just use `schema_version`. Let's look for `schema_version` first, and if that is not present, we look for `config_schema_version`. Users should be encouraged to use `schema_version`. Once all the current configs use this, we can remove support for `config_schema_version`.

Reply:
I'd rather change all to `schema_version` all at once. It'd be confusing to have two.

Reply:
Speaking to Rebecca yesterday, it seems we do not have to worry about backwards compatibility yet. Yay! So let's just use `schema_version` for now.